modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,45 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
 
-[NOTE]
-====
-Accessing Aura with SSO requires:
- 
-* Authorization Code Flow with PKCE.
-* A publicly accessible Identity Provider (IdP) server.
-====
+Aura admins can configure organization and tenant level SSO:
 
-== Console SSO
+. *Organization level SSO*: This configuration applies to all tenants and new database instances created within your organization.
```

Comment:
This is kinda true, but its not super clear. It prevents access to the org in console, not access to the actual database instance via bolt or http api or similar. So it only affects doing actions within console such as listing instances/pausing instances, editing tenant/org settings, etc. It applies to everything related to the console under this org.

Since theres a bit of overlap between doing instance operations in the console vs access the instance directly, do you think it would be best to clarify in some way the difference when we talk about org sso affecting console org access, console tenant access, and console instance access, from tenant level sso which affects instance access via bolt or similar, but not console?

Reply:
. *Organization level SSO*:
Applies to everything related to the console within an organization. Grants console access for organization management, this includes editing organization and tenant settings, viewing the list of instances within tenants, and managing instances (such as pause/resume). However, it does not grant access to view or modify data within the database instances themselves.

Is that more on the right track? @sampsonbryce

Reply:
Awesome - and I'll add a line at the end: "Currently, role mapping is not applicable to Organization level SSO."